### PR TITLE
feat(messagebox): add error,info and success message

### PR DESCRIPTION
**Pull request by lfbergee**
affects: @fremtind/jkl-message-box-react, @fremtind/jkl-message-box

## 📥 Proposed changes

style and react wrapper for messageboxes

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING]() doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

closes: #39

### CODE REVIEW

File: packages/message-box-react/src/MessageBox.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    title: string;
+    children: ReactNode;
+}
+
+type messageTypes = "info" | "error" | "success";
+
+function messageFactory(messageType: messageTypes) {
+    return function messageBox({ title, children }: Props) {
+        return (
+            <div className={`jkl-message-box__${messageType}`}>
```
Comment (piofinn) [inline packages/message-box-react/src/MessageBox.tsx]: Not BEM, should also have a "plain" `jkl-message-box`

File: packages/message-box/message-box.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    min-height: $layout-spacing--xl;
+    padding: $component-spacing--xl;
+}
+
+.jkl-message-box {
+    &__title {
+        margin-bottom: $component-spacing--large;
+    }
+    &__info {
+        @include box($snøhvit);
```
Comment (piofinn) [inline packages/message-box/message-box.scss]: This generates some redundant rules: Only `background-color` changes between the variants, but all properties are repeated.

File: packages/message-box/message-box.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        &::before {
+            @include colored-line($himmel);
```
Comment (piofinn) [inline packages/message-box/message-box.scss]: Same as above

**Comment by piofinn** [comment]
None of the above are _super_ important if we just want to get this in before the core refactor, but they should be addressed at some point (maybe make an issue if merged as is)

File: packages/message-box-react/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# `message-box-react`
```
[the remark this answers is not in the document]
Comment (lfbergee) [reply packages/message-box-react/README.md]: Added in style package

File: packages/message-box-react/src/MessageBox.tsx (hunk 1 above)
Comment (lfbergee) [reply packages/message-box-react/src/MessageBox.tsx]: Bemified it

File: packages/message-box/message-box.scss (hunk 2 above)
Comment (lfbergee) [reply packages/message-box/message-box.scss]: fixed

**Pull request merged**